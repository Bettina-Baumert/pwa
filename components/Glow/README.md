# Glow
---
The glow component is used to highlight elements that are being interacted with.

## Props

### color

_Default_: `colors.shade8`<br>
_Type_: `string`<br>

The color of the glow's background.

## Usage:

```markup
<Glow color="yellow">
  ...
</Glow>
```

### className

_Default_: `''`
_Type_: `string`

Additional classes that can be appended to the component's `className` attribute.

###### Usage:
```
<Glow className="class1 class2">
  ...
</Glow>
```

### styles

_Default_: `{ container: {}, glow: {} }`
_Type_: `Object`

An object of keys with styles relating to inner parts of the Glow.

###### Usage:
```
const styles = {
  container: {
    ...
  },
  glow: {
    ...
  }
};

<Glow styles={styles}>
  ...
</Glow>
```
