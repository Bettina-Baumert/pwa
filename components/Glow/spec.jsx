import React from 'react';
import { mount } from 'enzyme';
import Glow from './index';

describe('<Glow />', () => {
  it('should render with a smile', () => {
    const wrapper = mount(
      <Glow>
        <p>Glowing!</p>
      </Glow>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
