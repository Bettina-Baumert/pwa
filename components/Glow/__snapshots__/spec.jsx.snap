// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Glow /> should render with a smile 1`] = `
<Glow
  className={null}
  color="#f7f7f7"
  styles={
    Object {
      "container": null,
      "glow": null,
    }
  }
>
  <div
    aria-hidden={true}
    className="position-relative"
    onTouchTap={[Function]}
    style={null}
  >
    <div
      className="position-absolute left-0 top-0 w-100P h-100P b-transparent _1dnc6ae z-index-0"
      style={Object {}}
    />
    <p>
      Glowing!
    </p>
  </div>
</Glow>
`;
